examples/tfengine/modules/foundation.hcl

```diff
@@ -111,6 +114,25 @@ template "cicd" {
   recipe_path = "{{.recipes}}/cicd.hcl"
   output_path = "./cicd"
   data = {
+    resources = {
```

Comment:
I'm leaning towards hiding these resources inside the cicd recipe and not asking users to write them explicitly here as the purpose of cicd recipe is to expose a simpler interface for users to configure CICD pipelines. Resources here such as the IAM members are implementation details and should be handled automatically by the recipe.

Reply:
If we are automatically creating the service account with the required IAM members, then why don't we use the default cloudbuild service account?
The other concern I have is the `logs_bucket` creation. If we don't ask users for any inputs here, we have to pick a name for the bucket. Since the bucket name must be globally unique, this can potentially cause the CICD deployment to fail due to duplicate names.

Reply:
Implementation updated.